coderd/database/dbpurge/dbpurge.go

```diff
@@ -164,9 +183,20 @@ func New(ctx context.Context, logger slog.Logger, db database.Store, vals *coder
 				slog.F("duration", clk.Since(start)),
 			)
 
+			duration := clk.Since(start)
+			iterationDuration.WithLabelValues("true").Observe(duration.Seconds())
+			recordsPurged.WithLabelValues("expired_api_keys").Add(float64(expiredAPIKeys))
```

Comment:
This is great, but what about the other types? This gives the operator new information, but it's incomplete.